app/0.14.x/loadbalancing.md

```diff
@@ -49,6 +49,9 @@ Because there is no `weight` information, all entries will be treated as equally
 weighted in the load balancer, and the balancer will do a straight forward
 round-robin.
 
+Note that A records do not carry port information, so if you need Kong to route
+requests to specific port numbers, be sure to [configure Kong to ignore A records][dns-order-config].  
+
```

Comment:
DNS A records map names to IP addresses. DNS SRV records map service port numbers to names. Thus, in environments with services using ephemeral IP addresses and port numbers, Kong must obtain both DNS A and SRV records for routing purposes. In this case, I recommend additional research to determine the effects of ignoring DNS A records.

Reply:
This comment should be removed. Because often lookups are chained, eg. a name points to and SRV, which contains more names, which point to A records. In this case Kong will preserve the last port found, so it will take the port number from the SRV and the IP address from the A.

Reply:
Does the Kong resolver handle linking SRV records to A records or must the `dns_order` option include `SRV,A` to use both of them?